src/nems_subscription_create/external_integrations/pds/settings.py

```diff
@@ -0,0 +1,54 @@
+import base64
+import time
+import uuid
+from functools import lru_cache
+from typing import Mapping, Any
+
+from pydantic import BaseSettings, HttpUrl, validator
+
+
+class PDSSettings(BaseSettings):
+    jwt_rsa_private_key: str
+    jwt_sub: str
+    jwt_iss: str
+    jwt_aud: str = "https://int.api.service.nhs.uk/oauth2/token"
```

Comment:
I don't think we want to fall back to the TEST environment as a default if no ENV variables are found - if there's environment misconfiguration we should fail less gracefully as real patient information can't.

Reply:
Good point, I will explicitly fail here.